waspc/packages/deploy/src/providers/fly/deploy/deploy.ts

```diff
@@ -103,8 +104,11 @@ async function deployClient(deploymentInfo: DeploymentInfo<DeployOptions>, { bui
 	copyProjectClientTomlLocally(deploymentInfo.tomlFilePaths);
 
 	waspSays('Building web client for production...');
+	waspSays('If you configured a custom domain for the server, please run the command with an env variable: REACT_APP_API_URL=https://serverUrl.com wasp deploy fly deploy');
+
+	const serverUrl = stripTrailingSlash(process.env.REACT_APP_API_URL ?? '') || deploymentInfo.serverUrl;
```

Comment:
```suggestion
	const serverUrl = process.env.REACT_APP_API_URL
		? stripTrailingSlash(process.env.REACT_APP_API_URL)
		: deploymentInfo.serverUrl;
```
I'd do this to make the logic more explicit, right now there is `??` and `||` which have their own logic and this makes this a bit harder to read.

Reply:
That makes sense, definitely more readable with the ternary.